content/en/docs/getting-started.md

```diff
@@ -0,0 +1,249 @@
+namespace/volcano-system created
+configmap/volcano-scheduler-configmap created
+serviceaccount/volcano-scheduler created
+clusterrole.rbac.authorization.k8s.io/volcano-scheduler created
+clusterrolebinding.rbac.authorization.k8s.io/volcano-scheduler-role created
+deployment.apps/volcano-scheduler created
+serviceaccount/volcano-admission created
+clusterrole.rbac.authorization.k8s.io/volcano-admission created
+clusterrolebinding.rbac.authorization.k8s.io/volcano-admission-role created
+deployment.apps/volcano-admission created
+service/volcano-admission-service created
+job.batch/volcano-admission-init created
+serviceaccount/volcano-controllers created
+clusterrole.rbac.authorization.k8s.io/volcano-controllers created
+clusterrolebinding.rbac.authorization.k8s.io/volcano-controllers-role created
+deployment.apps/volcano-controllers created
+customresourcedefinition.apiextensions.k8s.io/jobs.batch.volcano.sh created
+customresourcedefinition.apiextensions.k8s.io/commands.bus.volcano.sh created
+customresourcedefinition.apiextensions.k8s.io/podgroups.scheduling.incubator.k8s.io created
+customresourcedefinition.apiextensions.k8s.io/queues.scheduling.incubator.k8s.io created
+customresourcedefinition.apiextensions.k8s.io/podgroups.scheduling.sigs.dev created
+customresourcedefinition.apiextensions.k8s.io/queues.scheduling.sigs.dev created
+
+```
+Verify the running components of Volcano
+```go
+# kubectl get all -n volcano-system
+NAME                                       READY   STATUS      RESTARTS   AGE
+pod/volcano-admission-5bd5756f79-p89tx     1/1     Running     0          6m10s
+pod/volcano-admission-init-d4dns           0/1     Completed   0          6m10s
+pod/volcano-controllers-687948d9c8-bd28m   1/1     Running     0          6m10s
+pod/volcano-scheduler-94998fc64-9df5g      1/1     Running     0          6m10s
+
+
+NAME                                TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)   AGE
+service/volcano-admission-service   ClusterIP   10.96.140.22   <none>        443/TCP   6m10s
+
+
+NAME                                  READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/volcano-admission     1/1     1            1           6m10s
+deployment.apps/volcano-controllers   1/1     1            1           6m10s
+deployment.apps/volcano-scheduler     1/1     1            1           6m10s
+
+NAME                                             DESIRED   CURRENT   READY   AGE
+replicaset.apps/volcano-admission-5bd5756f79     1         1         1       6m10s
+replicaset.apps/volcano-controllers-687948d9c8   1         1         1       6m10s
+replicaset.apps/volcano-scheduler-94998fc64      1         1         1       6m10s
+
+
+
+NAME                               COMPLETIONS   DURATION   AGE
+job.batch/volcano-admission-init   1/1           28s        6m10s
+
+```
+
+Volcano scheduler utilize `queues.scheduling.incubator.k8s.io` to share resource, therefore default queue is required before usage.
```

Comment:
Default queue now it's been handled in source code, therefore we don't need this step anymore.

Reply:
I have updated the document